docs/api/app.md

```diff
@@ -348,6 +348,25 @@ bar, and on OS X you can visit it from dock menu.
 
 Clears the recent documents list.
 
+### `app.setAsDefaultProtocolClient(protocol)` _OS X_ _Windows_
```

Comment:
On the other side, if you check your registry right now, you'll see that we (Microsoft) leave all uri schemes that we ever came across there. We really like to collect them ;-)

Buuuut for a @paulcbetts :+1:, I'd totally throw in a `removeAsDefaultProtocolHandler`, that checks if the current executable is set as handler for a given protocol and removes it. Thoughts?